src/homepageExperience/components/HomepageExperience.scss

```diff
@@ -110,3 +111,43 @@ h1#first-mile--header {
 .homepage-wizard--code-highlight {
   color: #b7b8ff;
 }
+
+/*
+ * Styles copied from the removed MePage.scss
```

Comment:
Is the reason we're keeping the styles that they get imported via the chronograf file - and they're used elsewhere in the app? I spot checked a couple of these and that seemed to be the case.

Reply:
correct. Other parts of the application depend on these styles, but the concept of a `MePage.scss` doesn't make sense anymore since `MePage.tsx` was removed